content/nic/install/helm/open-source.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Use Helm to Install NGINX Ingress Controller with NGINX Open Source
+linkTitle: NGINX Open Source
+toc: true
+weight: 100
+nd-content-type: how-to
+nd-product: INGRESS
+---
+
+This page describes how to use Helm to install F5 NGINX Ingress Controller with NGINX Open Source. 
+
+It explains the requirements for NGINX Ingress Controller, how to obtain and install the Helm chart, and what custom resource definitions (CRDs) are installed during the process.
+
+By following these instructions, you will finish with a functional NGINX Ingress Controller instance for your Kubernetes cluster.
+
+## Before you begin
+
+- A [supported Kubernetes version]({{< ref "/nic/technical-specifications.md#supported-kubernetes-versions" >}})
+- A functional Kubernetes cluster
+- [Helm 3.19+.](https://helm.sh/docs/intro/install)
+
+Throughout this page, you will see placeholder values indicated with angular brackets, such as **\<my-release\>**. Replace them accordingly for your installation.
+
+{{< call-out "warning" >}}
+
+The `edge` version **is not intended for production use**. It is intended for testing and development purposes only.
```

Comment:
minor: This warning feels somewhat aggressive to find at the top of this page, despite this use-case (installing `edge` version) not being the primary case most users likely need to consult this page for. Might it make sense to move this towards the bottom as a special case like the `Custom Resource Definitions` section in this doc?